apps/fulltextsearch.sh

```diff
@@ -27,7 +29,7 @@ root_check
 lowest_compatible_nc 18
 
 # Check if Full Text Search is already installed
-if ! does_this_docker_exist "$nc_fts" || ! is_app_installed fulltextsearch
+if ! does_this_docker_exist "$nc_fts" && ! does_this_docker_exist "$opens_fts" && ! is_app_installed fulltextsearch
```

Comment:
Sometimes the docker exists, but not the app. Sometimes the app exists, but not the docker. It should check for all ocurrances seperatly and ask to reinstall if any of them are true.

Reply:
Yes, that is what it is doing now. If any of the rules apply, it will start the reinstallation. Only if all are false, it will show the install screen  :)